monai/bundle/scripts.py

```diff
@@ -1539,3 +1539,126 @@ def init_bundle(
         copyfile(str(ckpt_file), str(models_dir / "model.pt"))
     elif network is not None:
         save_state(network, str(models_dir / "model.pt"))
+
+
+def _find_bundle_file(root_dir: Path, file_name: str, suffix: Sequence[str] = ("json", "yaml", "yml")) -> str | None:
+    # find bundle file with possible suffix
+    for _suffix in suffix:
+        full_name = f"{file_name}.{_suffix}"
+        if full_name in os.listdir(root_dir):
+            return full_name
+    return None
+
+
+class BundleManager:
+    """
+    The `BundleManager` class facilitates the automatic downloading and instantiation of bundles.
```

Comment:
will this support the python script based workflow? https://github.com/Project-MONAI/tutorials/tree/main/bundle/python_bundle_workflow cc @Nic-Ma

Reply:
I take a look at this one, it doesn't support it now, I will add some support for this one. Thanks.